jsk_perception/euslisp/detection_interface_servicecall.l

```diff
@@ -1,11 +1,12 @@
-(defun check-detection (&key (timeout 10) ((:type atype) nil) (tf-force nil)
-                             (single nil) (speak t)
-                             (speak-name)
-                             (detection-topic *detection-topic*)
-                             ((:target-object target-obj) (make-cube 60 60 60)))
+(defun check-detection-srv (&key (timeout 10) ((:type atype) nil)
```

Comment:
is this ok to change function name? do we need to update code which uses this function? @h-kamada